test/onnx/test_utility_funs.py

```diff
@@ -917,6 +919,11 @@ def forward(self, x, y):
                                (TestCase,),
                                dict(TestUtilityFuns.__dict__, opset_version=13))
 
+# opset 14 tests
+TestUtilityFuns_opset14 = type(str("TestUtilityFuns_opset14"),
+                               (TestCase,),
+                               dict(TestUtilityFuns.__dict__, opset_version=14))
+
 # opset 11 tests
 TestUtilityFuns_opset11_new_jit_API = type(str("TestUtilityFuns_opset11_new_jit_API"),
```

Comment:
nit: though unrelated, but seems these tests with name `new_jit_API` are no longer different from its above counterparts, can you remove them if that's the case?